### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by BranTkS**
## Description
basic footer for mental remedy

Fixes issue #13 

## How Has This Been Tested?
tested to  see if it changes affected code

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] Existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
+/* eslint-disable linebreak-style */
```
Comment (art-abdulwadud) [inline src/app.js:1]: ESLint errors can get frustrating sometimes but it's pretty easy to fix. The inline comment here works, but you would be cleaner and easier to add this rule to the `.eslintrc.js` file at the bottom where the other rules have already been created like this ```'linebreak-style': 0,```

Comment (BranTkS) [reply src/app.js:1]: alright fixed it

File: src/intro/Footer.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                  <h1 className="heading">Mental </h1>
+                </div>
+              </div>
+              {/* column 3 and a half*/}
+              <div className="col">
+                <div>
+                  <h1 className="heading">Remedy</h1>
+                </div>
+              </div>
+            </div>
```
Comment (art-abdulwadud) [inline src/intro/Footer.js:42]: Kindly, replace this section with a column of cities instead and make the footer responsive. You can personally reach out to me or other team members for any kind of assisstance regarding the footer.

File: src/index.css [hunk 3]
```diff
@@ -99,6 +105,48 @@ body {
...
+  width: 100%;
+}
+
+.list-unstyled {
+  list-style: none;
+}
+
 .large-btn {
   outline: none;
   border: 1px solid var(--bg-fallback);
```
Comment (Art-Abdullahi) [inline src/index.css:152]: kindly please move css styles for the footer to the `App.css`.  where all other styles for features are.

File: src/intro/Footer.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+export default class Footer extends Component {
+  render() {
+    return (
+      <div>
```
[the remark this answers is not in the document]
Comment (BranTkS) [reply src/intro/Footer.js]: edited it, should fix the issues

File: src/App.css [hunk 5]
```diff
@@ -435,3 +435,44 @@ input {
...
+  float: left;
+  font-size: var(--small-text);
+  font-family: var(--lato);
+  text-align: left;
+}
+
+.footer-container {
+  float: left;
+  width: 100%;
+}
```
[the remark this answers is not in the document]
Comment (BranTkS) [reply src/App.css]: thanks

File: src/intro/Footer.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                </ul>
+              </div>
+              {/* column 3*/}
+              <div className="col">
+                <ul className="list-unstyled">
+                  <h1 className="small-text">Cities</h1>
+                  <li>Nairobi </li>
+                  <li>Johannesburg </li>
+                </ul>
+              </div>
```
Comment (art-abdulwadud) [inline src/intro/Footer.js:37]: Nice work adding the new column

File: src/App.css [hunk 7]
```diff
@@ -435,3 +435,51 @@ input {
...
+  border-top: 3px solid aqua;
+  border-radius: 30%;
+}
+
+@media (max-width: 400px) {
+  .main-footer {
+    display: flex;
+    width: 100%;
+  }
+}
```
Comment (art-abdulwadud) [inline src/App.css:458]: You can use the already created media query instead ```@media (max-width: 480px){}```
respo2
try using css grid instead, it's not yet responsive on smaller devices

File: src/App.css [hunk 8]
```diff
@@ -434,4 +434,88 @@ input {
     --search-box-h: 3rem !important;
     --search-box-w: 90% !important;
   }
+
+  .col {
+    width: 25%;
+    grid-template-columns: auto;
+    grid-gap: 20px;
```
Comment (art-abdulwadud) [inline src/App.css:441]: Try using `grid-template-rows` as well, and also in the media queries. The whole point is to have every `.col` in its own row on smaller devices.
respo2 The footer still looks the same on smaller devices.

Comment (BranTkS) [reply src/App.css:441]: okay still working on it but it should be looking like this right now from the most recent push 
small screeen screenshot

**Pull request merged**